src/editors/containers/VideoEditor/components/VideoSettingsModal/components/VideoSourceWidget/hooks.jsx

```diff
@@ -0,0 +1,20 @@
+import { actions } from '../../../../../../data/redux';
+
+// export const state = {
+//   onFocusVideoId: (args) => React.useState(args),
+//   onFocus
+// };
+export const deleteFallbackVideo = ({ fallbackVideos, dispatch }) => (videoUrl) => {
```

Comment:
Could we maybe have some comments here, or perhaps somewhere better explaining what these inputs are, so the casual reader can understand? (docstring style would be best)